src/components/projects-summary/projects-summary.html

```diff
@@ -3,13 +3,8 @@
     <span class="fa fa-plus"></span>
     <span class="create-button-text">Create Project</span>
   </button>
-  <div ng-if="$ctrl.showGetStarted">
-    <h2 class="primary">Getting Started</h2>
-    <div class="getting-started-panel">
-      {{$ctrl.gettingStartedText}}
-    </div>
-  </div>
-  <h2 class="primary" ng-if="!$ctrl.showGetStarted">My Projects</h2>
+  <h2 class="primary" ng-if="!$ctrl.projects || !$ctrl.projects.length">Getting Started</h2>
+  <h2 class="primary" ng-if="$ctrl.projects && $ctrl.projects.length">My Projects</h2>
```

Comment:
Yeah, I like to not rely on angular being ok with referencing null objects' fields. Habit.